modules/sdk/src/main/scala/org/tessellation/sdk/domain/trust.scala

```diff
@@ -0,0 +1,30 @@
+package org.tessellation.sdk.domain
+
+import cats.syntax.bifunctor._
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust.{InternalTrustUpdate, InternalTrustUpdateRefined, InternalTrustUpdateRefinement}
+import org.tessellation.security.hex.Hex
+
+import derevo.cats.eqv
+import derevo.derive
+import eu.timepit.refined.cats._
+import eu.timepit.refined.refineV
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+import fs2.data.csv.{CellDecoder, DecoderError}
+
+object trust {
+  @derive(eqv)
+  case class TrustRating(peerId: PeerId, rating: InternalTrustUpdateRefined) {
+    def toInternalTrustUpdate = InternalTrustUpdate(peerId, rating)
+  }
+
+  object TrustRating {
+    implicit val peerIdDecoder: CellDecoder[PeerId] = CellDecoder.stringDecoder.map(Hex(_)).map(PeerId(_))
+
+    implicit val internalTrustUpdateRefinedDecoder: CellDecoder[InternalTrustUpdateRefined] =
+      CellDecoder.doubleDecoder.emap(refineV[InternalTrustUpdateRefinement](_).leftMap(new DecoderError(_)))
```

Comment:
I'm curious: Is there a reason these two decoders are not in their respective companion objects? `InternalTrustUpdateRefined` is being used by at least two types; if the behavior is intended to be the same in both cases then it feels like it should be its own type. WDYT?

Reply:
@sdavidp I think because they're cell decoders (for csv) so it will be used in this (and only in this) place.